### ISSUE TITLE
Refresh data post table migration

### ISSUE THREAD

**Issue by rymurr** [sundeck-io/OpsCenter#103]
After table schemas are changed (eg #101 ) we will have new columns in the reporting tables but the data from the backing tables (#88) or newly added calculated tables (#86) will be missing. Only newly arrived data will get the new fields. In this case we should add a banner to the main page which informs the user of new features being available and that they should go to the settings page and reset their data in order to pick up the new columns.

It is important that the banner is contextual and useful. Eg **NOT** "Go to settings -> reset" to get new data". Rather "In our latest release we added x,y,z (maybe a link). By resetting your data you will get these new features which will allow you to a,b,c (maybe link)"

**Comment by rymurr** [comment]
thinking it might be a good idea ot create an internal table with `VERSION, DESCRIPTION, AFFECTED_PAGES,REFRESH_TIMESTAMP`
We can write into this on release (with refresh_timestamp = `NULL`) and read from it when we showw a page. If hte page is in the table then we show the banner. When refresh is ticked insettings we update all null timestamps.

**Comment by joshelser** [comment]
Yeah, that's about what I was thinking. Thanks for writing it out!
